dune-project

```diff
@@ -25,6 +25,8 @@
    (minttea (>= "0.0.2"))
```

Comment:
Should this dep be removed?

Reply:
@yawaramin Thanks, great catch! Done in https://github.com/chshersh/github-tui/commit/51c23cea23a5177290d3696a1e39638fe5047c57

Appreciate you keeping an eye on this 🙂